connector/src/main/java/org/geysermc/connector/command/CommandSender.java

```diff
@@ -64,4 +68,30 @@ default String getLocale() {
      * @return true if the CommandSender has the requested permission, false if not
      */
     boolean hasPermission(String permission);
+
+    /**
+     * Get the GeyserSession of this {@link CommandSender}. Returns null if the Commandsender is console.
+     * Will only return a session if {@link GeyserConnector#getInstance()} does not return null,
+     * and {@link CommandSender#getName()} equals the username of a session.
+     *
+     * @return The corresponding GeyserSession, if it exists
+     */
+    @Nullable
+    default GeyserSession getGeyserSession() {
```

Comment:
```suggestion
    default GeyserSession asGeyserSession() {
```
Makes it more clear you’re transforming this object into a GeyserSession